### PR TITLE
Add auto instrumentation support to ansible

### PR DESCRIPTION
**Pull request by jeffreyc-splunk**

### CODE REVIEW

File: deployments/ansible/roles/collector/README.md [hunk 1]
```diff
@@ -137,3 +141,60 @@ how to use the role in a playbook with minimal required configuration:
...
   that nothing will be copied and existing `splunk_fluentd_config` will be used)
+
+### Auto Instrumentation
+
+- `install_splunk_otel_auto_instrumentation` (Linux only): Whether to
+  install/manage [Splunk OpenTelemetry Auto Instrumentation for Java](
+  https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation).
+  When set to `true`, the `splunk-otel-auto-instrumentation` deb/rpm package
+  will be downloaded and installed from the Collector repository. **Note:** The
+  Java application on the node need to be started/restarted separately after
```
Comment (pmcollins) [inline deployments/ansible/roles/collector/README.md:152]:
```suggestion
  Java application on the node needs to be started/restarted separately after
```

File: deployments/ansible/roles/collector/README.md [hunk 2]
```diff
@@ -137,3 +141,60 @@ how to use the role in a playbook with minimal required configuration:
...
+  (**default:** ``)
+
+- `splunk_otel_auto_instrumentation_service_name` (Linux only): Explicitly set
+  the [service name](
+  https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#configuration-file),
+  for the instrumented Java application, e.g. `my.service`. By default, the
+  service name is automatically derived from the arguments of the Java
+  executable on the node. The specifed service name will be added to the
+  `/usr/lib/splunk-instrumentation/instrumentation.conf` config file on the
+  node. **Note:** The Java application on the node needs to be restarted
```
Comment (pmcollins) [inline deployments/ansible/roles/collector/README.md:199]:
```suggestion
  node, overriding any generated service name. **Note:** The Java application on the node needs to be restarted
```

**Review by pmcollins — APPROVED** [review]
Just a couple of small text changes.

**Review by tmorrisonsfx — APPROVED** [review]

File: deployments/ansible/roles/collector/README.md [hunk 3]
```diff
@@ -137,3 +141,60 @@ how to use the role in a playbook with minimal required configuration:
...
+  file on the node. **Note:** The Java application on the node needs to be
+  restarted separately in order for any change to take effect.
+  (**default:** ``)
+
+- `splunk_otel_auto_instrumentation_service_name` (Linux only): Explicitly set
+  the [service name](
+  https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation#configuration-file),
+  for the instrumented Java application, e.g. `my.service`. By default, the
+  service name is automatically derived from the arguments of the Java
+  executable on the node. The specifed service name will be added to the
```
Comment (tmorrisonsfx) [inline deployments/ansible/roles/collector/README.md:197]: Typo: change ``specifed`` to "specified``

**Pull request merged**